src/sentry/options/defaults.py

```diff
@@ -1125,7 +1129,7 @@
 )
 register(
     "sentry-metrics.cardinality-limiter-rh.orgs-rollout-rate",
-    default=0.0,
+    default=1.0,
```

Comment:
This is intentionally at 0.0 for release health. A while back, we tried to enable it for the release health pipeline and it led to bottlenecks and impacted the pipeline's ability to process messages at a reasonable rate. I'd say let's keep it off for now since this incident follow-up is focused only on the generic metrics pipeline.